### DISCUSSION TITLE
Issue with hook for commits with paths that contain files with spaces

### DISCUSSION THREAD

**Discussion by wando777** [alirezanet/Husky.Net#107] [Q&A] [answered]
Hello, there 👋 

If the path to my staged directory contains spaces, when running any task, despite identifying the path with double quotes, I always get the return of `fatal: pathspec ... did not match any files`.
Here's my `task-runner.json`
```
{
  "tasks": [
    {
      "name": "echo staged files",
      "pathMode": "absolute",
      "command": "cmd",
      "group": "pre-commit",
      "args": [
        "/c",
        "echo",
        "${staged}"
      ]
    }
  ]
}
```

Here's also the output from my console after running `dotnet husky run`
```
[Husky] ? Preparing task 'echo staged files'
[Husky] ? Executing task 'echo staged files' ...
.editorconfig .config/dotnet-tools.json .husky/pre-commit .husky/task-runner.json "src/Private Assemblies/path-to-my-controller/MyController.cs"
fatal: pathspec 'src/Private' did not match any files
[Husky]  ? Successfully executed in 45m
```
So, even though it executes dotnet-format, the hook doesn't end up with the modified file in the staged area but the hook stops before that.

Any idea what's going on?

**Comment by alirezanet** [comment]
Hi @wando777,
unfortunately, I'm a little bit busy this week but I'll look into this the next week.

**Comment by wando777** [comment]
Hey @alirezanet,
Thank you for your previous response. I don't know if you've had an opportunity to review it yet, but I wanted to kindly ask. 😄 
Thank you once again!!

**Reply by alirezanet** [reply]
Hi @wando777,
sorry for the late reply, I opened a PR for this #111. I'll let you know if it solves the problem

**Reply by alirezanet** [reply, marked as answer]
Please test `v0.6.5` and let me know if it solved the problem.

**Reply by alirezanet** [reply]
Hi @wando777,
Did you test it? please confirm so I can create a release for this version

**Reply by wando777** [reply]
Thank you, @alirezanet , it worked! 👍
